gtsam/hybrid/HybridBayesNet.cpp

```diff
@@ -318,33 +291,28 @@ HybridValues HybridBayesNet::sample() const {
 }
 
 /* ************************************************************************* */
-double HybridBayesNet::error(const HybridValues &values) const {
-  GaussianBayesNet gbn = choose(values.discrete());
-  return gbn.error(values.continuous());
-}
-
-/* ************************************************************************* */
-AlgebraicDecisionTree<Key> HybridBayesNet::error(
+AlgebraicDecisionTree<Key> HybridBayesNet::logProbability(
     const VectorValues &continuousValues) const {
   AlgebraicDecisionTree<Key> error_tree(0.0);
```

Comment:
Rename `error_tree` to `log_prob_tree`?

Reply:
Sure

Reply:
Renamed to `result`. Note that naming convention for variables in gtsam is camlCase.